java-security/src/test/resources/x509Base64.txt

```diff
@@ -0,0 +1,35 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
is it the same formatting like you got it from vcap_services?

Reply:
file name and formatting changed according to received cert from x-forwarded-client-cert header